### PR TITLE
Update workflow files

### LINKED ISSUE #43: Add `cargo-deny` checks at PR level

### PR DESCRIPTION
**Pull request by yozhgoor**
- [X] Add `cargo-deny` for each PR and not only in pre-release.
- [X] Add the `package.yml` file in the repository.

Closes #43

### CODE REVIEW

**Comment by yozhgoor** [comment]
@adriantombu Do you see any issue if we add the `package.yml` file here?

**Comment by adriantombu** [comment]
It should be ok, the sensitive information are hidden as secrets on each repository ☺️

Feel free to also add this line in the `rust.yml` file like I did here https://github.com/ystorian/ally/blob/main/.github/workflows/rust.yml#L53C5-L53C25

```
needs: ubuntu-stable # Only run if the ubuntu-stable job succeeded
```

**Comment by yozhgoor** [comment]
I think we removed something previously because the all checks are stopped if one of them is not successful but this line change a bit the behavior and instead of starting every checks at the same time, you have to wait for Linux to succeed.
I don't mind adding it tho, maybe @cyann can pick for us?

**Comment by cyann** [comment]
I think this was a good idea to first run it with Linux, then run in parallel with the remaining targets. It's a bit slower (2~3 minutes) but this saves a lot of GitHub minutes.

Note that we will get GitHub Enterprise for free in 2 weeks (we're still in trial), we may get way more free minutes. If that changes, we could switch to do all targets in parallel.

[About billing for GitHub Actions](https://docs.github.com/en/billing/managing-billing-for-github-actions/about-billing-for-github-actions)
- Currently: 3'000 minutes/month
- Future: 50'000 minutes/month?

**Comment by yozhgoor** [comment]
It's updated, I only need a quick check for the `deny.toml` file

**Comment by yozhgoor** [comment]
@ystorian/template
8n1zxu

File: deny.toml [hunk 1]
```diff
@@ -13,4 +13,20 @@ allow = [
 	"CC0-1.0",
 	"BSD-3-Clause",
```
[the remark this answers is not in the document]
Comment (yozhgoor) [reply deny.toml:14]: Added. Don't worry I'm gonna refresh all the repositories that needs it so I can catch more détails like these

**Comment by cyann** [comment]
@adriantombu I've added you to the @ystorian/template team

**Pull request merged**